core/src/main/java/com/orange/ouds/core/theme/OudsTheme.kt

```diff
@@ -210,12 +213,16 @@ fun OudsThemeTweak(tweak: OudsTheme.Tweak, content: @Composable () -> Unit) {
 }
 
 /**
- * This function is equivalent to [isSystemInDarkTheme] except it takes the OUDS theme setting into account instead of the system one.
+ * Determines if the OUDS theme is currently in dark mode.
  *
- * The OUDS theme can be inverted or forced to light or dark when calling [OudsThemeTweak] and the value returned by this method reflects that kind of change.
- * If there is no call to [OudsThemeTweak] anywhere in the layout hierarchy, then this function returns the same value as [isSystemInDarkTheme].
+ * This function is equivalent to [isSystemInDarkTheme], but it respects overrides applied by [OudsThemeTweak].
  *
- * @return `true` if OUDS is considered to be in 'dark theme'.
+ * The OUDS theme can be inverted or forced to light/dark modes using [OudsThemeTweak].
+ * The value returned by this function reflects these effective changes.
+ *
+ * If [OudsThemeTweak] is not used in the layout hierarchy, this function returns the same value as [isSystemInDarkTheme].
```

Comment:
I realized that this sentence is not true. If `OudsThemeTweak` is not used, then this returns the value of the `darkThemeEnabled` parameter of the root `OudsTheme` method, which is not always equal to `isSystemInDarkTheme`.